integrated_channels/sap_success_factors/transmitters/content_metadata.py

```diff
@@ -19,6 +21,20 @@ def __init__(self, enterprise_configuration, client=SAPSuccessFactorsAPIClient):
             client=client
         )
 
+    def _filter_api_response(self, response, content_id):
+        """
+        Filter the response from SAPSF to only include the content
+        based on the content_id
+        """
+        try:
+            parsed_response = json.loads(response)
+            parsed_response["ocnCourses"] = [item for item in parsed_response["ocnCourses"]
+                                             if item["courseID"] == content_id]
+            filtered_response = json.dumps(parsed_response)
+            return filtered_response
```

Comment:
It would be good if we can capture the exception and log here.

Reply:
@zamanafzal "LOGGER.error("Error filtering response from SAPSF: %s", exc)" is this good enough for logging?

Reply:
Any reason we are not using `LOGGER.exception`?

Reply:
My bad, fixed it.